### ISSUE TITLE
Sorting rows

### ISSUE THREAD

**Issue by piernik** [a5hik/ng-sortable#160]
Your plugin converts to `display:block` - even `tr` row. 
Here is workaroud:
```
tr.as-sortable-item, tr.as-sortable-placeholder {
    display: table-row;
}
```

**Comment by deslauriersp** [comment]
@piernik : thanks for the workaround, works fine for me.
Dear @a5hik,

Thanks for the great ng-sortable! Just started using it.
Many people seems to have to the problem with tables rows, because of the "display:block"

https://github.com/a5hik/ng-sortable/issues/21 (you say it *was* removed)
https://github.com/a5hik/ng-sortable/issues/84 (you say you *can* remove it)

yet...current release 1.3.0  https://github.com/a5hik/ng-sortable/blob/master/dist/ng-sortable.css
The display block *is* there, at line 5 to 7.

I am a bit confused.  

Phil

Phil



Unless I am confused

**Comment by a5hik** [comment]
Hi actually this style is required and this is placed back via #112

**Comment by a5hik** [comment]
this is fixed in #237
